gamepad/gamepad-secure-context.html

```diff
@@ -5,21 +5,13 @@
 <script src="/resources/testharnessreport.js"></script>
 <script>
   test(() => {
-    // To avoid breaking legacy apps getGamepads() should always be exposed on
-    // the navigator object, even in non-secure contexts.
     assert_true("getGamepads" in Navigator.prototype, "in prototype");
     assert_true("getGamepads" in navigator, "on instance");
+    assert_array_equals(navigator.getGamepads(), []);
 
-    // Calling navigator.getGamepads() should return an empty array even if
-    // there are connected gamepads.
-    assert_array_equals(navigator.getGamepads(), [],
-                        "getGamepads returns an empty array");
-
-    // Gamepad API types should not be exposed in non-secure contexts.
-    assert_equals(typeof(Gamepad), "undefined", "Gamepad type is undefined");
-    assert_equals(typeof(GamepadButton), "undefined",
-                  "GamepadButton type is undefined");
-    assert_equals(typeof(GamepadEvent), "undefined",
-                  "GamepadEvent type is undefined");
-  }, "Gamepad API is not exposed in non-secure contexts");
+    assert_true("Gamepad" in window, "Gamepad in window");
+    assert_true("GamepadButton" in window, "GamepadButton in window");
+    assert_true("GamepadEvent" in window, "GamepadButton in window");
+    assert_true("GamepadHapticActuator" in window, "GamepadButton in window");
+  }, "Gamepad API and interfaces are exposed in non-secure contexts");
```

Comment:
What do you think about deleting this file entirely?

Reply:
yeah... I was thinking the same... [idlharness.window.js](https://github.com/web-platform-tests/wpt/pull/44691/files#diff-a788f3dd179ecc15a731b46a982ce5ca0a737a67e92402a85a459dee75507ea7) does the same thing.